recipes/orcania/all/conandata.yml

```diff
@@ -0,0 +1,12 @@
+sources:
+  "2.3.1":
+    url: "https://github.com/babelouest/orcania/archive/refs/tags/v2.3.1.tar.gz"
+    sha256: "bbf08d563528b8ab88dd4b0e67aeb4e7c4fc9f19dcd1a0346b773cf492f7612b"
+patches:
+  "2.3.1":
+    - patch_file: "patches/2.3.1-0001-shared-static-conan.patch"
```

Comment:
this should also be upstreamed 🙏 ❤️

Reply:
https://github.com/babelouest/orcania/issues/31